spec/requests/host_meta_request_spec.rb

```diff
@@ -1,14 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Everything that was in this file was also handled by the controller spec, and thus is now included in the newly created request spec.